src/SprykerEco/Shared/Payone/PayoneConstants.php

```diff
@@ -12,6 +12,9 @@
  */
 interface PayoneConstants
 {
+    public const ROUTE_LOGIN = 'login';
```

Comment:
Please read whats written above in docblock:

    Declares global environment configuration keys. Do not use it for other class constants.
    
As such, these constants are invalid to be used here.
Please put them into the Config files instead.